### PR TITLE
Cosmetic Updates

### PR DESCRIPTION
**Pull request by kevinsun-dev**
## Overview

Minor cosmetic changes to bring the UI closer in line to the Figma spec.

## Changes Made

- Replaced settings emoji with ic_settings icon from Figma.
- Added ic_plus_sign icon for adding polls integration.
- General font-family fixes across 4 activites.

## Next Steps (delete if not applicable)

Ensuring that things don't break across different DPIs, ensuring dynamic UI changes for longer and shorter text.

## Screenshots

Screenshot_20190926-220351_Pollo

### CODE REVIEW

**Comment by ningning621** [comment]
omg samsung's emojis are so cute 
image

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -40,20 +42,26 @@
...
                     app:layout_constraintVertical_bias="0.66" />
 
-                <TextView
-                    android:id="@+id/textView6"
-                    android:layout_height="wrap_content"
-                    android:layout_marginStart="16dp"
-                    android:layout_marginTop="12dp"
-                    android:layout_width="wrap_content"
+                <ImageView
+                    android:id="@+id/imageView2"
```
Comment (ningning621) [inline app/src/main/res/layout/activity_main.xml]: lmao these names will have to change later (not a priority rn)

**Comment by ningning621** [comment]
> LGTM--A bunch of the sizes/fonts/spacing (mostly old and a few of the ones you changed) are inconsistent with the new designs, but imo it would make more sense just to convert everything to what's on Zeplin in its own PR. Would probably need to do that page by page for the sake of having reasonably sized PRs

we can do an audit once a v.0 is finished...unless someone wants to take charge of auditing the ui rn

**Comment by kevinsun-dev** [comment]
> LGTM--A bunch of the sizes/fonts/spacing (mostly old and a few of the ones you changed) are inconsistent with the new designs, but imo it would make more sense just to convert everything to what's on Zeplin in its own PR. Would probably need to do that page by page for the sake of having reasonably sized PRs

Zeplin/Figma run differently than how Android studio does. Many measurements there work under the assumptions that the screen is pixel perfect to the 360x640 screens that Design has mocked up, but for Android that's just not the case. I needed to use a 10dp spacing on one side, and 16dp on the other, to remotely imitate what Figma says to be 14px spacings. To the letter works for iOS, but for a platform as variable and intricate as Android, the best and only way might be "to the spirit".

> we can do an audit once a v.0 is finished...unless someone wants to take charge of auditing the ui rn

Since I'm working a bit more on-off on Pollo than the both of them, I've decided to take command of handling the old UI and perhaps scripting up the new. I'll be releasing more UI patches over the next few weeks.

File: app/src/main/res/layout/group_list_item.xml [hunk 2]
```diff
@@ -2,65 +2,63 @@
...
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+        android:id="@+id/groupLiveTextView"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignTop="@id/groupNameTextView"
+        android:layout_marginStart="18dp"
+        android:layout_marginTop="2dp"
+        android:textColor="#9EA7B3"
```
Comment (young-k) [inline app/src/main/res/layout/group_list_item.xml]: should we have these defined in a `colors.xml` or something of the sort? (and the other hex colors as well)

Comment (ningning621) [reply app/src/main/res/layout/group_list_item.xml]: ideally we should. @kevinsun-dev i think we can start moving the hex to `colors.xml` and confirming that they're 1:1 with the colors on figma as you audit each screen.

**Pull request merged**